src/types.js

```diff
@@ -112,9 +112,9 @@ export type UserCredentials = {
 
 export type Device = DeviceAttributes & {
   connected: boolean,
-  functions?: Array<string>,
+  functions?: ?Array<string>,
```

Comment:
1) ` we have device object without functions and vars when return devices list. 
https://github.com/Brewskey/spark-server/blob/dev/src/managers/DeviceManager.js#L125-L129
2)  functions and vars can be nullable when we return details:
https://github.com/Brewskey/spark-server/blob/dev/src/managers/DeviceManager.js#L111-L114
That why i do `key?: ?type` here